### PR TITLE
compat: Dask 2024.11

### PR DESCRIPTION
**Pull request by ahuang11**
Hopefully?

> ERROR geoviews/tests/test_projection.py - FutureWarning: The legacy Dask DataFrame implementation is deprecated and will be removed in a future version. Set the configuration option `dataframe.query-planning` to `True` or None to enable the new Dask Dataframe implementation and silence this warning.

Taken from https://github.com/holoviz/holoviews/pull/6149/files

### CODE REVIEW

**Comment by hoxbro** [comment]
It is the new dask release, try to pin 2024.10 for now. I will look into it soon.

**Comment by hoxbro** [comment]
FYI https://github.com/holoviz/geoviews/pull/763 (if you haven't seen it)

File: geoviews/tests/conftest.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 from contextlib import suppress
 
+from holoviews.tests.conftest import bokeh_backend, port, serve_hv  # noqa: F401
+
```
Comment (hoxbro) [inline geoviews/tests/conftest.py:4]:
```suggestion
from holoviews.tests.conftest import bokeh_backend, port, serve_hv  # noqa: F401

```

File: pixi.toml [hunk 2]
```diff
@@ -102,6 +102,8 @@ rioxarray = "*"
 scipy = "*"
 shapely = "*"
 xarray = "*"
+dask-core = "*"
+dask-expr = "*"
```
Comment (hoxbro) [inline pixi.toml:106]:
```suggestion
```

Dask is not a direct dependency of GeoViews

File: pyproject.toml [hunk 3]
```diff
@@ -175,9 +175,9 @@ filterwarnings = [
     "ignore:\\s*Pyarrow will become a required dependency of pandas:DeprecationWarning", # Will go away by itself in Pandas 3.0
     "ignore:Passing a (SingleBlockManager|BlockManager) to (Series|GeoSeries|DataFrame|GeoDataFrame) is deprecated:DeprecationWarning", # https://github.com/holoviz/spatialpandas/issues/137
     "ignore:datetime.datetime.utcfromtimestamp():DeprecationWarning:dateutil.tz.tz", # https://github.com/dateutil/dateutil/pull/1285
-    # 2024-02
-    "ignore:The current Dask DataFrame implementation is deprecated:DeprecationWarning", # https://github.com/dask/dask/issues/10917
     # 2024-03
     "ignore:\\s*Dask dataframe query planning is disabled because dask-expr is not installed:FutureWarning",
     "ignore:numpy.ndarray size changed, may indicate binary incompatibility:RuntimeWarning", # https://github.com/pydata/xarray/issues/7259
+    # 2024-11
+    "ignore:The legacy Dask DataFrame implementation is deprecated and will be removed in a future version.:FutureWarning", # https://github.com/dask/dask/issues/10917
```
Comment (hoxbro) [inline pyproject.toml:182]: Can you remove the dask stuff from the contest file and pixi.toml file? I think they were only there when Datashader did not directly support `dask-expr`.

https://github.com/holoviz/geoviews/blob/79cde9d60d15bafc1ad7621486caf4ee3450f7ab/geoviews/tests/conftest.py#L52-L57

https://github.com/holoviz/geoviews/blob/79cde9d60d15bafc1ad7621486caf4ee3450f7ab/pixi.toml#L13

File: geoviews/tests/conftest.py (hunk 1 above)
Comment (ahuang11) [reply geoviews/tests/conftest.py:4]: Was this accidental?

Comment (hoxbro) [reply geoviews/tests/conftest.py:4]: Yes (I updated it just now). I don't think these changes are related to this PR.

Comment (ahuang11) [reply geoviews/tests/conftest.py]: Ah I see :) leaked from the other PR.

File: pyproject.toml (hunk 3 above)
Comment (hoxbro) [inline pyproject.toml:182]: Is this still needed?

**Review by hoxbro — APPROVED** [review]
One small comment but LGTM.

**Pull request merged**